tests/testthat/_snaps/db-io.md

```diff
@@ -0,0 +1,28 @@
+# db_copy_to() wraps DBI errors
+
+    Code
+      (expect_error(db_copy_to(con = con, table = "tmp", values = data.frame(x = c(1,
+        1)))))
+    Output
+      <error/rlang_error>
+      Error in `db_copy_to()`:
+      ! Can't copy to table "tmp"
+      Caused by error in `dplyr::db_write_table()`:
+      ! Can't write table "tmp".
+      Caused by error:
+      ! table 'tmp' already exists
+
+---
+
+    Code
+      (expect_error(db_copy_to(con = con, table = "tmp2", values = data.frame(x = c(1,
+        1)), unique_indexes = list("x"))))
+    Output
+      <error/rlang_error>
+      Error in `db_copy_to()`:
+      ! Can't copy to table "tmp2"
+      Caused by error in `db_create_index.DBIConnection()`:
```

Comment:
I'm a bit confused that this mentions the method and above it is the generic. But that seems to be something related to rlang and probably not really worth to fix.

Reply:
Yeah, I wouldn't worry about it either.